types/Config.js

```diff
@@ -21,6 +21,34 @@ export type HasteConfig = {|
 
 export type ConfigGlobals = Object;
 
+export type GlobalConfig = {
+  bail: boolean,
+  collectCoverage: boolean,
+  collectCoverageFrom: Array<Glob>,
+  collectCoverageOnlyFrom: {[key: string]: boolean},
+  coverageDirectory: string,
+  coveragePathIgnorePatterns: Array<string>,
+  coverageReporters: Array<string>,
+  coverageThreshold: {global: {[key: string]: number}},
```

Comment:
as i said in the morning meeting, we need more granular thresholds. Even the coverage for one project as a whole doesn't make any sense most of the time (unless it's a standalone library or something very small)